pkg/kubelet/dockershim/helpers_linux.go

```diff
@@ -103,12 +104,17 @@ func (ds *dockerService) updateCreateConfig(
 		// TODO: Can we assume the defaults are sane?
 		rOpts := lc.GetResources()
 		if rOpts != nil {
+			max_pids := config.Linux.MaxPids
+			if max_pids == 0 {
+				max_pids = ds.dockerPIDsLimit
```

Comment:
I feel like this should be Min(PodMaxPids, PIDsLimit)?

Reply:
agree, makes sense